### PR TITLE
DOCSP-33172 added meta description for top 250 pages in docs-compass

### PR DESCRIPTION
**Pull request by sarahemlin**
## DESCRIPTION
Hey @sarah-olson-mongodb!  I'm adding descriptions to the top 250 pages that don't have them, and here are the Compass pages that needed them.  I mostly just copied from the page text, but please check for coherence & conventions and edit as you see fit.

Thanks!
Sarah

## STAGING
https://preview-mongodbsarahemlin.gatsbyjs.io/compass/DOCSP-33172/

## JIRA
https://jira.mongodb.org/browse/DOCSP-33172

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=655cf2445b686d6bd98a3c30

### CODE REVIEW

File: source/collections.txt [hunk 1]
```diff
@@ -10,6 +10,9 @@ Collections
    :name: genre
    :values: tutorial
 
+.. meta:: 
+   :description: How to create or drop a collection, which is a grouping of MongoDB documents equivalent to a table in a relational database system.
```
Comment (sarah-olson-mongodb) [inline source/collections.txt:14]: (suggestion)
Consider dropping the phrase explaining collections. So just something like: 
"How to use Compass to create or drop a MongoDB collection." 

My reasoning is that the collection definition might be too much detail for a summary page.

File: source/index.txt [hunk 2]
```diff
@@ -8,6 +8,9 @@
 What is MongoDB Compass?
 ========================
 
+.. meta:: 
+   :description: MongoDB Compass is a powerful GUI for querying, aggregating, and analyzing your MongoDB data in a visual environment run on macOS, Windows, and Linux.
```
Comment (sarah-olson-mongodb) [inline source/index.txt:12]:
```suggestion
.. meta:: 
   :description: MongoDB Compass is a GUI for querying, aggregating, and analyzing your data in a visual environment run on macOS, Windows, and Linux.
```
Minor suggestion! Not critical.

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks @sarahemlin. Couple minor suggestions for consideration. Over all LGTM! 

One thing I noticed is that the structure of the description varies in a few ways.
- How to ... 
Ex: How to add and view favorite saved queries and aggregations.

- Noun ... verb
Ex: MongoDB Compass is a powerful GUI for querying, aggregating, and analyzing your MongoDB data in a visual environment run on macOS, Windows, and Linux.

- Verb ... noun
Ex: Query documents in your Atlas, Enterprise, or Community database deployments.

It might be useful to develop some guidance on these descriptions so the team ca. take up this work, going forward.

File: source/collections.txt (hunk 1 above)
Comment (sarahemlin) [reply source/collections.txt:14]: Ian mentioned putting 'table' in and said there wasn't information anywhere that told the user that tables are collections for us.  I thought that was a good idea to make sure that we've got that additional way to access it.  What do you think?

**Pull request merged**